Manual.md

```diff
@@ -734,6 +734,12 @@ The current list of available `build_style` scripts is the following:
 `do_configure()`, `do_build()`, etc., and may overwrite default `do_fetch()` and
 `do_extract()` that fetch and extract files defined in `distfiles` variable.
 
+- `cargo` For packages written in rust that use Cargo for building. Configuration
+arguments (such as `--features`) can be passed to cargo in `do_build` via
+`configure_args`. Sets the `RUST_TARGET` variable, which can be useful for
```

Comment:
Hm, how would you write it? I basically only want to express that one uses `configure_args` to configure the build (and as such they're used in `do_build` instead of the usual `do_configure`)

Reply:
(Disclaimer: Not a native speaker)
> Configuration arguments (such as `--features ...`) can be defined via `configure_args` which is applied as command line arguments to cargo during  `do_build`.

Reply:
Thanks, that does sound better, I'll use something along the lines of that. Did you have a chance to test any of the cross compiled binaries yet?

Reply:
Changed it, please take a 2nd look at it.